pupil_src/shared_modules/recorder.py

```diff
@@ -286,8 +286,9 @@ def start(self):
             frame = self.g_pool.capture._recent_frame
             if frame is None:
                 logger.error(
-                    "Recording a Pupil Mobile stream requires a connection!"
-                    " Aborting recording."
+                    "Recording a Pupil Mobile stream requires a connection. "
+                    "Aborting recording. "
+                    "Try turning down camera resolution and/or frame rate."
```

Comment:
After discussing this message in a meeting, we have decided on this wording:
```suggestion
                    "Your connection does not seem to be stable enough for "
                    "recording Pupil Mobile via WiFi. We recommend recording "
                    "on the phone."
```